### PR TITLE
Add activation functions in jax.nn module

### PR DESCRIPTION
**Pull request by karalleyna**
**JAX Frontend**

- [x] sigmoid(x)
- [x] softplus(x)
- [x] soft_sign(x)
- [x] silu(x) 
- [x] swish(x) 
- [x] log_sigmoid(x) 
- [x] leaky_relu(x[, negative_slope])
- [x] hard_sigmoid(x)
- [x] hard_silu(x) 
- [x] hard_swish(x)
- [x] hard_tanh(x) 
- [x] elu(x[, alpha])
- [x] celu(x[, alpha]) 
- [x] selu(x)
- [x] glu(x)

### CODE REVIEW

**Review by fspyridakos — CHANGES_REQUESTED** [review]
Great PR, just some suggeted changes, please remove duplicates in particular and update the docstrings. I'll take another look once you're done. Thanks!

File: ivy/functional/frontends/torch/nn/functional/non_linear_activation_functions.py [hunk 1]
```diff
@@ -283,21 +249,15 @@ def leaky_relu_(input, negative_slope=0.01):
...
 
 
 @to_ivy_arrays_and_back
 def hardsigmoid(input, inplace=False):
-    ret = ivy.divide(ivy.minimum(ivy.maximum(ivy.add(input, 3), 0), 6), 6)
-    if inplace:
-        ivy.inplace_update(input, ret)
-        return input
-    return ret
+    # ret = ivy.divide(ivy.minimum(ivy.maximum(ivy.add(input, 3), 0), 6), 6)
```
[the remark this answers is not in the document]
Comment (karalleyna) [reply ivy/functional/frontends/torch/nn/functional/non_linear_activation_functions.py]: Done

File: ivy/functional/ivy/experimental/activations.py [hunk 2]
```diff
@@ -284,3 +285,689 @@ def batch_norm(
...
+
+
+@integer_arrays_to_float
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+@handle_array_like_without_promotion
+@handle_array_function
+def sigmoid(
```
[the remark this answers is not in the document]
Comment (karalleyna) [reply ivy/functional/ivy/experimental/activations.py]: Removed

File: ivy/functional/ivy/experimental/activations.py [hunk 3]
```diff
@@ -284,3 +285,689 @@ def batch_norm(
...
+    return current_backend(x).log_sigmoid(x, out=out)
+
+
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+@handle_array_like_without_promotion
+@handle_array_function
+def softplus(
```
[the remark this answers is not in the document]
Comment (karalleyna) [reply ivy/functional/ivy/experimental/activations.py]: Removed

File: ivy/functional/ivy/experimental/activations.py [hunk 4]
```diff
@@ -284,3 +285,689 @@ def batch_norm(
...
+    return current_backend(x).hard_silu(x, out=out)
+
+
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+@handle_array_like_without_promotion
+@handle_array_function
+def leaky_relu(
```
[the remark this answers is not in the document]
Comment (karalleyna) [reply ivy/functional/ivy/experimental/activations.py]: Removed

**Comment by AnnaTz** [comment]
Hi @karalleyna, could you please re-make this PR? It can't get merged as it is now because of the force-push.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
